fe-nc-games/src/components/IndividualReviewDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+function IndividualReviewDetails({
+  review_id,
```

Comment:
I'm nitpicking _a bit_, but could you change your components so they're written as `const IndividualReviewDetails = () => {}` rather than the function syntax? It's just more of a standard way of doing things, and helps differentiate between functions that are in the same component later on 👍

Reply:
Sure, will do, thanks John.